README.md

```diff
@@ -13,18 +13,27 @@ The SPIFFE Helper is a simple utility for fetching X.509 SVID certificates from
 
 If `-config` is not specified, the default value `helper.conf` is assumed. 
 
-The flag `-exitWhenReady` is also supported.
+CLI options:
+
+ | Flag name                       | Description                                                         |
+ |---------------------------------|---------------------------------------------------------------------|
+ | `-config`                       | Path to the configuration file                                      |
+ | `-help`                         | Print interactive help                                              |
+ | `-daemon-mode={true|false}`     | Overrides `daemon_mode` in the config file.                         |
+ | `-exitWhenReady`                | Obsolete and ignored. Use `-daemon-mode=false` instead.             |
 
 ## Configuration
+
 The configuration file is an [HCL](https://github.com/hashicorp/hcl) formatted file that defines the following configurations:
 
  | Configuration                       | Description                                                                                                                       | Example Value                                                                                                                                                        |
  |-------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------|
  | `agent_address`                     | Socket address of SPIRE Agent.                                                                                                    | `"/tmp/agent.sock"`                                                                                                                                                  |
  | `cmd`                               | The path to the process to launch.                                                                                                | `"ghostunnel"`                                                                                                                                                       |
- | `cmd_args`                          | The arguments of the process to launch.                                                                                           | `"server --listen localhost:8002 --target localhost:8001--keystore certs/svid_key.pem --cacert certs/svid_bundle.pem --allow-uri-san spiffe://example.org/Database"` |
+ | `cmd_args`                          | The arguments of the process to launch and (in daemon mode) monitor and signal for certificate renewals.                          | `"server --listen localhost:8002 --target localhost:8001--keystore certs/svid_key.pem --cacert certs/svid_bundle.pem --allow-uri-san spiffe://example.org/Database"` |
```

Comment:
I think the part about monitoring and signaling would work better as a description for cmd

Reply:
Updated